plugins/lua/buildingplan/planneroverlay.lua

```diff
@@ -345,11 +345,13 @@ function PlannerOverlay:init()
     local main_panel = widgets.Panel{
         view_id='main',
         frame={t=1, l=0, r=0, h=14},
-        frame_style=gui.INTERIOR_MEDIUM_FRAME,
+        frame_style=gui.FRAME_MEDIUM,
         frame_background=gui.CLEAR_PEN,
         visible=self:callback('is_not_minimized'),
     }
 
+    main_panel.frame_style.signature_pen = false
```

Comment:
as written, this is changing the `signature_pen` for every tool that uses `gui.FRAME_MEDIUM`, which is probably not what you want. Option 1 would be to use `copyall(gui.FRAME_MEDIUM)` and alter that copy. Option 2 would be to use `gui.FRAME_INTERIOR_MEDIUM` that already has the change. I'd go with option 2 and keep the `FRAME_INTERIOR_MEDIUM` in `gui`.

Reply:
> as written, this is changing the `signature_pen` for every tool that uses `gui.FRAME_MEDIUM`, 

oh god no °-° no, i didn't mean that at all, of course!
crap, i didn't even notice this. 🤦 i didn't think about checking other MEDIUM frames tbh.

dammit! I always forget all these intricacies of Lua. I mean-- i presumed this would only affect the "copy" that planneroverlay is using, but i see how that might not really be what is happening D:

yeah no ok, we should probably go back to using INTERIOR_MEDIUM. I'll edit that back in the next update to the PR